doc/api/process.md

```diff
@@ -1819,6 +1819,46 @@ process.report.writeReport();
 
 Additional documentation is available in the [report documentation][].
 
+## process.resourcesUsage()
+<!-- YAML
+added: v12.x.x
+-->
+
+* Returns: {Object}
```

Comment:
How did you decide what to expose and what to omit from http://docs.libuv.org/en/v1.x/misc.html#c.uv_rusage_t

Reply:
I removed heap/cpu related fields since they are exposed throught `process.cpuUsage()` and `process.memoryUsage()`. Other fields i removed are supported on neither linux or windows (see https://linux.die.net/man/2/getrusage description for each field)

Reply:
I have a mild preference for exposing the fields as-is, supported or not, duplicating other APIs or not.

If that obsoletes `process.cpuUsage()`, then so be it. You could reimplement it as a JS-only shim around `process.resourceUsage()`.

(`process.memoryUsage()` still has merit because it also reports the V8 heap size.)